src/logic/hydra.lean

```diff
@@ -33,7 +34,7 @@ namespace relation
 
 open multiset prod
 
-variables {α β : Type*}
+variable {α : Type*}
```

Comment:
I always advise against using `variable`. I personally always get confused when getting a cryptic error due to the missing `s`
```suggestion
variables {α : Type*}
```